Simplified/NYPLBookCellDelegate.m

```diff
@@ -183,10 +183,19 @@ - (void)openAudiobook:(NYPLBook *)book {
     dict[@"id"] = book.identifier;
   }
   
+  id<DRMDecryptor> audiobookDrmDecryptor = nil;
+
+  if ([book.distributor isEqualToString:LCPAudiobooks.distributorKey]) {
```

Comment:
this check is repeated many times (3 in MyBooksDownloadCenter, 1 here) and it seems like a key part of the business logic to identify a LCP audiobook. It would be worth it to have this as a static function in LCPAudiobooks, like `func verifyBookIsLCP(_ book: NYPLBook)`. This would also be more future-proof in case we need to expand its semantics in the future.

Reply:
a good idea, added `canOpenBook` to `LCPAudiobooks`